docs/socketexample.rst

```diff
@@ -222,15 +223,26 @@ to know about all the adapters that convert a German to a US socket type:
    >>> sockets = list(zope.component.getAdapters((bathroomDE,), IUSSocket))
    >>> len(sockets)
    3
-   >>> names = [name for name, socket in sockets]
-   >>> names.sort()
+   >>> names = sorted([str(name) for name, socket in sockets])
    >>> names
-   [u'', u'dvd', u'shaver']
+   ['', 'dvd', 'shaver']
 
-`zope.component.getAdapters()` returns a list of tuples. The first
+:func:`zope.component.getAdapters` returns a list of tuples. The first
 entry of the tuple is the name of the adapter and the second is the
 adapter itself.
 
+Note that the names are ``unicode`` on Python 2:
+
+.. doctest::
+
+   >>> try:
+   ...    unicode = unicode
+   ... except NameError:
+   ...    unicode = str
+   >>> [isinstance(name, unicode) for name, _ in sockets]
+   [True, True, True]
+
+
```

Comment:
Seems redundant given the example above, but if we included this, also use `text`.